full-moon/src/visitors.rs

```diff
@@ -302,20 +302,26 @@ create_visitor!(ast: {
     }
 
     // Lua 5.2
-    #[cfg(any(feature = "lua52", feature = "luajit"))] {
+    #[cfg(any(feature = "lua52", feature = "luajit", feature = "cfxlua"))] {
         visit_goto => Goto,
         visit_label => Label,
     }
 
     #[cfg(feature = "lua54")] {
         visit_attribute => Attribute,
     }
+
+    #[cfg(all(feature = "cfxlua", not(feature = "luau")))] {
```

Comment:
I don't quite understand this condition.

We can remove the `visit_compound_assignment` from the Luau section above and then use `any`

```suggestion
    #[cfg(any(feature = "cfxlua", feature = "luau"))] {
```